src/components/full-screen-player/index.tsx

```
@@ -395,6 +395,9 @@ const FullScreenPlayer = () => {
                   <MusicPlayProgress className="w-full" trackClassName="h-[6px]" />
                   <div className="flex w-full items-center justify-center space-x-4">
                     <MusicPlayMode />
+                      <IconButton title="桌面歌词" onPress={() => window.electron.toggleDesktopLyrics()}>
```

ESLint

[error] Delete `··`

(prettier/prettier)